moveit_ros/warehouse/warehouse/src/warehouse_services.cpp

```diff
@@ -133,12 +133,14 @@ int main(int argc, char** argv)
   int port;
   node.param<std::string>("warehouse_host", host, "localhost");
   node.param<int>("warehouse_port", port, 33829);
+
+  ros::Duration(10.0).sleep();
```

Comment:
Is this sleep actually needed here if you increased the timeout *that much* below?

Reply:
Not really needed. But when trying to connect it keeps displaying a connection warning so I thought it might be useful to avoid that being printed from the beginning. But I could remove it if you think is unnecessary.

Reply:
in that case please remove it here.